helm/datadog_checks/helm/helm.py

```diff
@@ -0,0 +1,79 @@
+import base64
+import json
+from gzip import decompress
+
+from kubernetes import client, config
+
+from datadog_checks.base import AgentCheck
+
+# Values to mirror https://github.com/sstarcher/helm-exporter/blob/master/main.go#L50
+# Makes for easier migration
+STATUS_MAP = {
+    "unknown": 0,
+    "deployed": 1,
+    "uninstalled": 2,
+    "superseded": 3,
+    "failed": -1,
+    "uninstalling": 5,
+    "pending-install": 6,
+    "pending-upgrade": 7,
+    "pending-rollback": 8,
+}
+
+
+class HelmCheck(AgentCheck):
+    __NAMESPACE__ = 'helm'
+
+    def __init__(self, *args, **kwargs):
+        config.load_incluster_config()
+        self.v1 = client.CoreV1Api()
+        AgentCheck.__init__(self, *args, **kwargs)
+
+    def check(self, instance):
+        # The command kubectl  get secret -l owner=helm --field-selector 'type=helm.sh/release.v1' -A
+        all_releases = self.v1.list_secret_for_all_namespaces(label_selector='owner=helm')
```

Comment:
can we add the field selector to the query?

Reply:
The field selector isn't actually part of the helm go code, so I don't include it here. I can update the documentation.